server/models/postgis/tags.py

```diff
@@ -53,3 +53,12 @@ def get_all_campaigns():
         dto = TagsDTO()
         dto.tags = [r for r, in result]
         return dto
+
+    def delete(self):
+        """ Delete the user in scope from DB """
+        db.session.delete(self)
+        db.session.commit()
+
+    @staticmethod
+    def get_by_organisation_tag(organisation_tag):
+        return Tags.query.filter_by(organisations = organisation_tag).one_or_none()
```

Comment:
If we have true duplicates and two tags match a string, I think this will error saying two found when one expected.  Possibly worth writing a test for this.

Reply:
yes i think of this too. but when i see the method used to insert organisation tag, i see that it takes care of the duplicate (it wont add the organisation if it already exist). do you have any idea to further prevent it?

Reply:
@fiennyangeln in that case I'm happy to use the string